README.md

```diff
@@ -40,25 +40,20 @@
   </a>
 </p>
 
-Open Autonomy is a framework for the creation of Agent Services: off-chain services which run as a multi-agent-system (MAS)  and are replicated on a temporary consensus gadget (blockchain) while being crypto-economically secured on a public blockchain, hence offering robustness, transparency and decentralization off-chain.
+**Open Autonomy** is a framework for the creation of **agent services**: off-chain
```

Comment:
I think it should be `autonomous service` in this case

Reply:
@DavidMinarsch please, give your decision, as it seems there is a missunderstanding here, and I will adhere to it consistently. See also the naming convention document.

Reply:
Let's go with the table as discussed @jmoreira-valory

Reply:
(as in with the naming convention document)

Reply:
Ok, then 'agent services'.